docs/Changelog.md

```diff
@@ -20366,6 +20600,50 @@ This version of the operator has been available since version 16 of the default
 <dd>Constrain types to int tensors.</dd>
 </dl>
 
+### <a name="STFT-16"></a>**STFT-16**</a>
+
+  Computes the Short-time Fourier Transform of the signal.
+
+#### Version
+
+This version of the operator has been available since version 16 of the default ONNX operator set.
+
+#### Attributes
+
+<dl>
+<dt><tt>onesided</tt> : int (default is 0)</dt>
+<dd>If onesided is 1, only values for w in [0, 1, 2, ..., floor(n_fft/2) + 1] are returned because the real-to-complex Fourier transform satisfies the conjugate symmetry, i.e., X[m, w] = X[m,w]=X[m,n_fft-w]*. Note if the input or window tensors are complex, then onesided output is not possible. Enabling onesided with real inputs performs a Real-valued fast Fourier transform (RFFT).When invoked with real or complex valued input, the default value is 0. Values can be 0 or 1.</dd>
```

Comment:
(nit) Space after "(RFFT).When".

Reply:
https://github.com/smk2007/onnx/pull/1

Reply:
Update in the code, this will be generated at the end and reflected in this file.